portal-api/src/v1/roles/validate-user-has-at-least-one-allowed-role.js

```diff
@@ -9,9 +9,14 @@ const { userHasAtLeastOneAllowedRole } = require('./user-has-at-least-one-allowe
 const validateUserHasAtLeastOneAllowedRole = ({
   allowedRoles,
 }) => (req, res, next) => {
+
   const { user } = req;
 
-  if (!userHasAtLeastOneAllowedRole({ user, allowedRoles })) {
+  if (!user) {
+    throw new Error(`Failed to create user session with status ${allowedRoles}: user is null`);
```

Comment:
Please add `console.error`.

Reply:
Added `console.error`.

Reply:
Console.error to use string substitituion.

Reply:
Added string substitituion to console.error